samcli/lib/providers/sam_function_provider.py

```diff
@@ -108,7 +108,7 @@ def _extract_functions(stacks: List[Stack], ignore_code_extraction_warnings=Fals
             Function configuration object
         """
 
-        result = {}
+        result = {}  # dict: full_path -> Function
```

Comment:
no types?

Reply:
Sorry about the confusion, I meant to explain what are the keys and values. Updated